bookingTour/src/main/java/com/naite/bookingTour/controller/BookingController.java

```diff
@@ -14,69 +9,51 @@
 import org.springframework.web.bind.annotation.ModelAttribute;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.SessionAttributes;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
 import com.naite.bookingTour.model.Booking;
 import com.naite.bookingTour.model.Tour;
+import com.naite.bookingTour.model.User;
 import com.naite.bookingTour.repository.BookingRepository;
-import com.naite.bookingTour.repository.ClientRepository;
-import com.naite.bookingTour.repository.TourRepository;
+import com.naite.bookingTour.repository.UserRepository;
+import com.naite.bookingTour.service.TourService;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpSession;
 
 @Controller
-@RequestMapping("/booking")
 public class BookingController {
+	@Autowired
+	private UserRepository userRepository;
+	
+	@Autowired
+	private TourService tourService;
+
+	@GetMapping("/booking/{tourId}")
```

Comment:
```suggestion
	@GetMapping("/bookings/{tourId}")
```
tour id nó nên là paramater chứ ko phải là path vairable bé nha
cơ mà thôi kệ